packages/sharing-smart-contract/contracts/registry/AppWhitelist.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
+
+/******************************************************************************
+ * Copyright 2020 IEXEC BLOCKCHAIN TECH                                       *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License");            *
+ * you may not use this file except in compliance with the License.           *
+ * You may obtain a copy of the License at                                    *
+ *                                                                            *
+ *     http://www.apache.org/licenses/LICENSE-2.0                             *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ ******************************************************************************/
+pragma solidity ^0.8.23;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "../interfaces/IDataProtectorSharing.sol";
+import "../interfaces/IAppWhitelist.sol";
+import "../interfaces/IRegistry.sol";
+
+contract AppWhitelist is IAppWhitelist, Ownable {
+    // ---------------------AppWhitelist state------------------------------------
+    IProtectedDataSharing internal immutable _protectedDataSharing;
```

Comment:
should it be a simple address? `address internal immutable _appsOwner`

Reply:
yeah it could. I find that more explicit. But i can change. May be i will wait for Hadrien review on this part